config/electronBuilder/electronBuilder.local.config.js

```diff
@@ -1,15 +1,23 @@
-const ENVIRONMENT = require('../src/CONST/ENVIRONMENT');
+const ENVIRONMENT = require('../../src/CONST/ENVIRONMENT');
 
+const isStagingBuild = process.env.NODE_ENV === 'staging';
```

Comment:
```suggestion
const isStaging = process.env.NODE_ENV === 'staging';
```
"build" just doesn't really need to be in the name, and it's really not a build but more of an environment.